src/reducers/entities/search.js

```diff
@@ -171,4 +220,13 @@ export default combineReducers({
     // Object where every key is a team composed with
     // an object where the key is the term and the value indicates is "or" search
     recent,
+
+    // Object holding the current searches for every team
+    current,
+
+    // Boolean true if we are are searching initally
+    isSearchingTerm,
```

Comment:
Do we need both isSearchingTerm and isSearchGettingMore since they're opposites of each other? 0/5 though

Reply:
They are not opposites. They are both false if nothing is happening.

Reply:
Ah right. I missed that part